src/detours.cpp

```diff
@@ -910,12 +910,13 @@ struct _DETOUR_TRAMPOLINE
     _DETOUR_ALIGN   rAlign[8];      // instruction alignment array.
     PBYTE           pbRemain;       // first instruction after moved code. [free list]
     PBYTE           pbDetour;       // first instruction of detour function.
+    ULONG64         rbIndTarget;
 };
 
-C_ASSERT(sizeof(_DETOUR_TRAMPOLINE) == 184);
+C_ASSERT(sizeof(_DETOUR_TRAMPOLINE) == 192);
```

Comment:
I think, part of the point, but not the main point, is this can now be reduced.

Reply:
Oh, maybe not. The large increase is not the jmp per se, but the other copied instructions?